src/lang/english.txt

```diff
@@ -453,6 +452,11 @@ STR_SETTINGS_MENU_SANDBOX_OPTIONS                               :Sandbox options
 STR_SETTINGS_MENU_TRANSPARENCY_OPTIONS                          :Transparency options
 STR_SETTINGS_MENU_TOWN_NAMES_DISPLAYED                          :Town names displayed
 STR_SETTINGS_MENU_STATION_NAMES_DISPLAYED                       :Station names displayed
+STR_SETTINGS_MENU_STATION_NAMES_TRAIN                           :{TRAIN}  Train stations
+STR_SETTINGS_MENU_STATION_NAMES_LORRY                           :{LORRY}  Lorry stops
+STR_SETTINGS_MENU_STATION_NAMES_BUS                             :{BUS}  Bus stops
+STR_SETTINGS_MENU_STATION_NAMES_SHIP                            :{SHIP}  Docks
+STR_SETTINGS_MENU_STATION_NAMES_PLANE                           :{PLANE}  Airports
```

Comment:
Should there be "two" spaces between vehicle icon codes and string?

Reply:
So, you're saying it is needed for now. OK checked.

Reply:
#13322 will remove it, it was a mistake to do this on my part.